openadapt/events.py

```diff
@@ -364,33 +411,32 @@ def get_merged_events(to_merge, state):
                         "mouse_x": event.mouse_x,
                         "mouse_y": event.mouse_y,
                         "mouse_button_name": event.mouse_button_name,
-                        "children": [
-                            event,
-                            t_to_event[release_t],
-                        ],
+                        "children": [event, t_to_event[release_t]],
                     },
                 )
             event.timestamp -= state["dt"]
             merged.append(event)
         return merged
 
-
     return merge_consecutive_action_events(
-        "mouse_click", events, is_target_event, get_merged_events,
+        "mouse_click",
+        events,
+        is_target_event,
+        get_merged_events,
     )
 
 
-def merge_consecutive_keyboard_events(events, group_named_keys=True):
-    """Merge consecutive keyboard char press events into a single press event"""
+def merge_consecutive_keyboard_events(
+    events: Any, group_named_keys: bool = True
```

Comment:
`events: list[ActionEvent]`

`group_named_keys: bool = KEYBOARD_EVENTS_MERGE_GROUP_NAMED_KEYS`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507